### PR TITLE
Clarify the doc that we use OpenLineage for metadata collection.

### PR DESCRIPTION
**Pull request by fm100**
Signed-off-by: Minkyu Park <minkyu@datakin.com>

### Problem
Write only client is deprecated in favor of OpenLineage, but the document is still mentioning about Marquez's metadata API for data collection. This PR makes it clear we use OpenLineage for collection while we keep the metadata API for querying them.

### Solution
Clarified the documentation.

### Checklist

- [x] You've [signed-off](https://github.com/MarquezProject/marquez/blob/main/CONTRIBUTING.md#sign-your-work) your work
- [ ] Your changes are accompanied by tests (_if relevant_)
- [x] Your change contains a [small diff](https://kurtisnusbaum.medium.com/stacked-diffs-keeping-phabricator-diffs-small-d9964f4dcfa6) and is self-contained
- [ ] You've updated any relevant documentation (_if relevant_)
- [ ] You've updated the [`CHANGELOG.md`](https://github.com/MarquezProject/marquez/blob/main/CHANGELOG.md#unreleased) with details about your change under the "Unreleased" section (_if relevant, depending on the change, this may not be necessary_)
- [ ] You've versioned your `.sql` database schema migration according to [Flyway's naming convention](https://flywaydb.org/documentation/concepts/migrations#naming) (_if relevant_)

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -45,17 +45,11 @@ Marquez enables highly flexible [data lineage](https://en.wikipedia.org/wiki/Dat
...
+* **Metadata API**: RESTful API enabling a diverse set of clients to begin interacting with metadata around dataset production and consumption.
 * **Metadata UI**: Used for dataset discovery, connecting multiple datasets and exploring their dependency graph.
 
 <br/>
 
-<figure align="center">
-  <img src="./assets/images/design.png">
-</figure>
-
-To ease adoption and enable a diverse set of data processing applications to build metadata collection as a core requirement into their design, Marquez provides language-specific clients that implement the [Metadata API](./openapi.html). As part of our initial release, we have provided support for [Java](https://github.com/MarquezProject/marquez-java) and [Python](https://github.com/MarquezProject/marquez-python).
```
Comment (collado-mike) [inline docs/index.md:57]: Rather than deleting this paragraph, I would reword it to state that we provide python/java clients for _reading_ metadata.

File: docs/index.md [hunk 2]
```diff
@@ -45,17 +45,11 @@ Marquez enables highly flexible [data lineage](https://en.wikipedia.org/wiki/Dat
...
 
 * **Metadata Repository**: Stores all job and dataset metadata, including a complete history of job runs and job-level statistics (i.e. total runs, average runtimes, success/failures, etc).
-* **Metadata API**: RESTful API enabling a diverse set of clients to begin collecting metadata around dataset production and consumption.
+* **Metadata API**: RESTful API enabling a diverse set of clients to begin interacting with metadata around dataset production and consumption.
 * **Metadata UI**: Used for dataset discovery, connecting multiple datasets and exploring their dependency graph.
 
 <br/>
 
-<figure align="center">
-  <img src="./assets/images/design.png">
```
Comment (collado-mike) [inline docs/index.md:54]: I don't know how this image was generated, but it would be great if we can just alter the image to specify that the clients are OpenLineage clients, but then keep the rest.

File: docs/index.md [hunk 3]
```diff
@@ -45,16 +45,16 @@ Marquez enables highly flexible [data lineage](https://en.wikipedia.org/wiki/Dat
...
 * **Metadata Repository**: Stores all job and dataset metadata, including a complete history of job runs and job-level statistics (i.e. total runs, average runtimes, success/failures, etc).
-* **Metadata API**: RESTful API enabling a diverse set of clients to begin collecting metadata around dataset production and consumption.
+* **Metadata API**: RESTful API enabling a diverse set of clients to begin interacting with metadata around dataset production and consumption.
 * **Metadata UI**: Used for dataset discovery, connecting multiple datasets and exploring their dependency graph.
 
 <br/>
 
 <figure align="center">
-  <img src="./assets/images/design.png">
+  <img src="./assets/images/ol-stack.svg">
```
Comment (fm100) [inline docs/index.md:54]: Brought this image from OpenLineage.
